contracts/OffsetHelper.sol

```diff
@@ -439,9 +431,6 @@ contract OffsetHelper is OffsetHelperStorage {
         if (amounts[0] < amountIn) {
             IERC20(_fromToken).approve(dexRouterAddress, 0);
         }
-
-        // update balances
-        balances[msg.sender][_poolToken] += _toAmount;
     }
```

Comment:
Shouldn't we return `amounts[0]` (also need to update the `swapExactOutToken` function signature to return a `uint256`) to make `swapExactOutToken` more composable?